src/main/webapp/college-info.js

```diff
@@ -0,0 +1,335 @@
+  // Cost Section.
+  const costDiv = document.getElementById('cost');
+  costDiv.innerHTML = '';
+  costDiv.append(`In-State Tuition: $${inStateTuition}`);
+  costDiv.append(`Out-of-State Tuition: $${outOfStateTuition}`);
+
+  // Admissions Section.
+  const admissionsDiv = document.getElementById('admissions');
+  admissionsDiv.innerHTML = '';
+  admissionsDiv.append(`Acceptance Rate: ${acceptanceRate}%`);
+  admissionsDiv.append(`Average SAT Score: ${avgSat}`);
+  admissionsDiv.append(`Average ACT Score: ${avgAct}`);
+
+  // Students Section.
+  const studentsDiv = document.getElementById('students');
+  studentsDiv.innerHTML = '';
+  studentsDiv.append(`Population: ${numStudents} Students`);
+  studentsDiv.append(`4 Year Graduation Rate: ${graduationRate4yr}%`);
+
+  // Draw charts.
+  chartsPromise.then(() => {
+    drawRaceChart(numWhiteStudents, numAsianStudents, numBlackStudents,
+        numHispanicStudents, numIndigenousStudents, numMultiracialStudents,
+        numUnreportedRaceStudents);
+    drawGenderChart(numMen, numWomen);
+  });
+}
+
+/**
+ * Creates Donut Pie Chart displaying racial breakdown.
+ * @param {number} numWhiteStudents
+ * @param {number} numAsianStudents
+ * @param {number} numBlackStudents
+ * @param {number} numHispanicStudents
+ * @param {number} numIndigenousStudents
+ * @param {number} numMultiracialStudents
+ * @param {number} numUnreportedRaceStudents
+ */
+function drawRaceChart(numWhiteStudents, numAsianStudents, numBlackStudents,
+    numHispanicStudents, numIndigenousStudents, numMultiracialStudents,
+    numUnreportedRaceStudents) {
+  const data = google.visualization.arrayToDataTable([
+    ['Race', 'Percentage'],
+    ['White', numWhiteStudents],
+    ['Asian', numAsianStudents],
+    ['Black', numBlackStudents],
+    ['Hispanic', numHispanicStudents],
+    ['Indigenous Ameircan/Alaskan', numIndigenousStudents],
+    ['Two or More Races', numMultiracialStudents],
+    ['Unreported', numUnreportedRaceStudents],
+  ]);
+
+  const options = {
+    title: 'Breakdown by Race',
+    pieHole: 0.4,
+    colors: ['#C6ACA4', '#A4C5C6', '#FFEB99',
+      '#856C8B', '#C6BDA4', '#D4EBD0', '#C68B77'],
+  };
+
+  const chart = new google.visualization.PieChart(document
+      .getElementById('race-piechart'));
+  chart.draw(data, options);
+}
+
+/**
+ * Creates Donut Pie Chart displaying gender breakdown.
+ * @param {number} numMen
+ * @param {number} numWomen
+ */
+function drawGenderChart(numMen, numWomen) {
+  const data = google.visualization.arrayToDataTable([
+    ['Gender', 'Percentage'],
+    ['Men', numMen],
+    ['Women', numWomen],
+  ]);
+
+  const options = {
+    title: 'Breakdown by Gender',
+    pieHole: 0.4,
+    colors: ['#D4EBD0', '#A4C5C6'],
+  };
+
+  const chart = new google.visualization.PieChart(document
+      .getElementById('gender-piechart'));
+  chart.draw(data, options);
+}
+
+/**
+ * Finds and returns the main campus from a list of main and satellite campuses
+ *     of the same name.
```

Comment:
This line actually shouldn't indent, interestingly enough. Yes indent params, no indent descriptions.